packages/runtime/src/context.ts

```diff
@@ -77,26 +67,10 @@ export type Context = {
     content: Content,
     options?: CreateOpts,
   ) => Promise<ModelInstanceDocument<Content>>
-  /**
-   * Update an existing document.
-   */
-  updateDoc: <Content extends Record<string, any>>(
```

Comment:
why are some of these methods removed from this class, but not others?

Reply:
I used to treat the loader as a purely internal API so the context would be the primary interface to interact with Ceramic, but now that the loader is extracted to a public package I think it's OK to use the `context.loader` directly from the GraphQL schema handlers.
These functions on the context were just proxies to the loader calls without any additional logic, so I think it's better to remove them and use the `context.loader` equivalents directly.

Reply:
but then why not push all these functions down from the context into the loader?  Why do some stay here and others not? What's different between the ones that move vs the ones that stay here?  Can we treat them all the same?

Reply:
The context has additional logic, such as checking if the Ceramic instance has an authenticated DID or not and using the authenticated DID as controller for example.
I'd rather keep the loader rather low-level with the need to explicitly set the controller so the loader doesn't need to handle authentication at all, and leaving it up to the context or other higher level consumers to handle the logic they need, what do you think?